src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -27,20 +28,22 @@
 
     //Data fields of Patient
     protected final String age;
+    protected final String priority;
 
     /**
      * Constructs a {@code JsonAdaptedPerson} with the given person details.
      */
     @JsonCreator
     public JsonAdaptedPerson(@JsonProperty("type") String type, @JsonProperty("nric") String nric,
             @JsonProperty("name") String name, @JsonProperty("phone") String phone,
-            @JsonProperty("age") String age) {
+            @JsonProperty("age") String age, @JsonProperty("priority") String priority) {
         this.type = type;
         this.nric = nric;
         this.name = name;
         this.phone = phone;
 
```

Comment:
small issue: maybe remove this extra line

Reply:
Removed.